tests/test_events.py

```diff
@@ -56,9 +55,6 @@ def get_trainer(self, **kwargs):
     @pytest.mark.parametrize('device,deepspeed_zero_stage', [
         pytest.param('cpu', None, id='cpu-ddp'),
         pytest.param('gpu', None, id='gpu-ddp', marks=pytest.mark.gpu),
-        pytest.param('gpu', 0, id='deepspeed-zero0', marks=pytest.mark.gpu),
```

Comment:
There is an issue with deepspeed here -- the tests pass, but somehow the errors overflow into other tests.